src/commercetools/testing/custom_objects.py

```diff
@@ -97,3 +106,19 @@ def get_by_container_key(self, request, container: str, key: str):
             return create_commercetools_response(request, json=item)
         else:
             return create_commercetools_response(request, status_code=404)
+
+    def update_by_container_key(self, request, container: str, key: str):
+        item = self.model._get_by_container_key(container, key)
+        return self._update(request, item)
+
+    def delete_by_container_key(self, request, container: str, key: str):
```

Comment:
Maybe we should name it `delete_by_container_and_key` to avoid users misreading it as deleting a bunch of objects by container key. But this depends on whether this naming convention is used elsewhere in a similar way.